libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -48,7 +48,7 @@
 <ng-template #empty>
   <div class="empty">
     <p>
-      Try searching for a topic, for example "<a (click)="searchExample()"
+      Try searching for a topic, for example "<a tabindex="0" role="link" (click)="searchExample()"
```

Comment:
These additional attributes aren't necessary, <a> elements already have this information baked in